app/decorators/offence_decorator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class OffenceDecorator < BaseDecorator
+  def plea_list
+    return t('generic.not_available') if pleas.blank?
+    return pleas unless pleas.is_a?(Enumerable)
+
+    safe_join(plea_sentences, tag(:br))
```

Comment:
I thought `tag(:br)` was legacy syntax, should it be `tag.br`

Reply:
`content_tag(:br, options)` is legacy.  I used `tag(:br)` because it creates one `<br/>` and for some reason `tag.br` creates two `<br/><br/>` - i need to double-check this

Reply:
amended that to `tag.br`, not sure what i did wrong before where it created two